src/main/java/org/mockito/Captor.java

```diff
@@ -15,9 +15,16 @@
  *
  *    &#64;Captor ArgumentCaptor&lt;AsyncCallback&lt;Foo&gt;&gt; captor;
  *
+ *    private AutoCloseable closeable;
+ *
  *    &#64;Before
- *    public void init(){
- *       MockitoAnnotations.initMocks(this);
+ *    public void open() {
+ *       MockitoAnnotations.openMocks(this);
```

Comment:
```suggestion
 *       closeable = MockitoAnnotations.openMocks(this);
```

Reply:
Awesome! We could also update Byte Buddy some time soon to get initial Java 16 support.

Reply:
Could you apply this suggestion?